CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## CHANGE LOG
 
+### Version 1.5.0 (March 4, 2022)
+* Supports [CleverTap Android SDK v4.4.0](https://github.com/CleverTap/clevertap-android-sdk/releases/tag/core-v4.4.0) 
```

Comment:
@william-ct Add that this release also supports iOS SDK v4.0.0 and give the link to this https://github.com/CleverTap/clevertap-ios-sdk/releases/tag/4.0.0